app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -121,4 +117,6 @@ class ScheduleAppointmentSheet : BottomSheetActivity() {
   fun enableDecrementButton(state: Boolean) {
     decrementDateButton.isEnabled = state
   }
+
+  data class ScheduleAppointment(val displayText: String, val timeAmount: Int, val chronoUnit: ChronoUnit)
```

Comment:
Can you move this out of the activity and into the package as a separate file?

Reply:
Since the scope of this class is limited to this activity, I'd prefer keeping it here.
We are doing this in other places as well. Refer `AppointmentReminderSheet`